pages/Login/index.tsx

```diff
@@ -19,8 +20,14 @@ const Login: NextPage = () => {
     formState: { errors },
   } = useForm<ILogin>();
   const router = useRouter();
+  const { data, mutate, ...dataQuery } = useLogin();
 
-  const onSubmit: SubmitHandler<ILogin> = (data) => router.push("/Home");
+  const onSubmit: SubmitHandler<ILogin> = (dataForm) => {
+    const success = (token: any) => {
+      router.push("/Home", token);
```

Comment:
all route names should be centralized in a file to make it easier to change a route name